build.sh

```diff
@@ -2,110 +2,18 @@
 
 set -e
 
-cd $(dirname $0)
+export ContinuousIntegrationBuild=true
+export Configuration=Release
 
-source toolversions.sh
+echo Building...
+dotnet build -nologo -clp:NoSummary -v quiet Google.Cloud.EntityFrameworkCore.Spanner
```

Comment:
I'd still expect to have to specify `-c $Configuration` here. (Whereas ContinuousIntegrationBuild is picked up automatically.)

Reply:
(And pack and test, of course)

Reply:
Closing the loop, it seems like [all MS Build properites may be specified through environment variables](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-properties?view=vs-2022#environment-properties). And running the script confirms it.